catalyst-gateway/Earthfile

```diff
@@ -139,4 +139,24 @@ test:
         --service event-db-is-running
         RUN --mount=$EARTHLY_RUST_CARGO_HOME_CACHE --mount=$EARTHLY_RUST_TARGET_CACHE \
             cargo nextest run --release --run-ignored=only signed_docs
-    END 
\ No newline at end of file
+    END
+
+test-scylla:
+    FROM +builder-src
+
+    # setup scylla
+    RUN apt-get update && \
+        apt-get install -y --no-install-recommends \
+        wget gnupg ca-certificates && \
+        mkdir -p /etc/apt/keyrings && \
+        gpg --homedir /tmp --no-default-keyring --keyring /etc/apt/keyrings/scylladb.gpg \
+            --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys a43e06657bac99e3 && \
+        wget -O /etc/apt/sources.list.d/scylla.list http://downloads.scylladb.com/deb/debian/scylla-6.2.list && \
```

Comment:
Can you make the scylla version that we pull as a earthly ARG? so we could run the same test with different scylla versions

Reply:
I made this as separate function one, so we can select the version from this.